interpreter/nodev8/v8.go

```diff
@@ -414,9 +425,20 @@ type v8Data struct {
 			Flags               uint16 `name:"flags__uint32_t"`
 		}
 
+		DeoptimizationData struct {
```

Comment:
Can you add a link to the origin of this struct? Similar to the other structs above?
Also please for `SourcePositionTable` and `SharedFunctionInfoWrapper`.